engine/apps/schedules/tests/test_ical_utils.py

```diff
@@ -93,17 +93,57 @@ def test_users_in_ical_email_case_insensitive(make_organization_and_user, make_u
 
 
 @pytest.mark.django_db
-def test_users_in_ical_viewers_inclusion(make_organization_and_user, make_user_for_organization):
+@pytest.mark.parametrize(
+    "role,included",
+    [
+        (LegacyAccessControlRole.ADMIN, True),
+        (LegacyAccessControlRole.EDITOR, True),
+        (LegacyAccessControlRole.VIEWER, False),
+        (LegacyAccessControlRole.NONE, False),
+    ],
+)
+def test_users_in_ical_basic_role(make_organization_and_user, make_user_for_organization, role, included):
+    organization, user = make_organization_and_user()
+    other_user = make_user_for_organization(organization, role=role)
+
+    usernames = [user.username, other_user.username]
+    expected_result = {user}
+    if included:
+        expected_result.add(other_user)
+
+    result = users_in_ical(usernames, organization)
+    assert set(result) == expected_result
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "permission,included",
+    [
+        (RBACPermission.Permissions.NOTIFICATIONS_READ, True),
+        (RBACPermission.Permissions.SCHEDULES_READ, False),
+        (None, False),
+    ],
+)
+def test_users_in_ical_rbac(make_organization_and_user, make_user_for_organization, permission, included):
     organization, user = make_organization_and_user()
+    organization.is_rbac_permissions_enabled = True
+    organization.save()
+
     viewer = make_user_for_organization(organization, role=LegacyAccessControlRole.VIEWER)
+    viewer.permissions = [{"action": permission.value}] if permission else []
+    viewer.save()
 
     usernames = [user.username, viewer.username]
+    expected_result = {user}
+    if included:
+        expected_result.add(viewer)
+
     result = users_in_ical(usernames, organization)
-    assert set(result) == {user}
+    assert set(result) == expected_result
 
 
 @pytest.mark.django_db
-def test_list_users_to_notify_from_ical_viewers_inclusion(
+def test_list_users_to_notify_from_ical_viewers_exclusion(
```

Comment:
changed name here.. if you check down on line 172 this is actually testing that viewers are _excluded_ (as would be expected by the "fallback role" when RBAC is disabled)